cfme/common/physical_server_views.py

```diff
@@ -0,0 +1,188 @@
+            data_dict['state'] = quad_data.xpath(self.QUADRANT.format(pos="b"))[0].get('style')
+            data_dict['vendor'] = quad_data.xpath(self.QUADRANT.format(pos="c"))[0].get('alt')
+            data_dict['creds'] = quad_data.xpath(self.QUADRANT.format(pos="d"))[0].get('alt')
+        return data_dict
+
+
+def PhysicalServerEntity():  # noqa
+    """ Temporary wrapper for PhysicalServer Entity during transition to JS based Entity
+
+    """
+    return VersionPick({
+        Version.lowest(): NonJSPhysicalServerEntity,
+        '5.9': JSPhysicalServerEntity,
+    })
+
+
+class PhysicalServerDetailsToolbar(View):
+    """Represents physical toolbar and its controls."""
+    monitoring = Dropdown(text="Monitoring")
+    configuration = Dropdown(text="Configuration")
+    policy = Dropdown(text="Policy")
+    power = Dropdown(text="Power")
+    identify = Dropdown(text="Identify")
+    lifecycle = Dropdown(text="Lifecycle")
+
+    @ParametrizedView.nested
+    class custom_button(ParametrizedView):  # noqa
+        PARAMETERS = ("button_group", )
+        _dropdown = Dropdown(text=Parameter("button_group"))
+
+        def item_select(self, button, handle_alert=False):
+            self._dropdown.item_select(button, handle_alert=handle_alert)
+
+
+class PhysicalServerDetailsEntities(View):
+    """Represents Details page."""
+    properties = SummaryTable(title="Properties")
+    networks = SummaryTable(title="Networks")
+    relationships = SummaryTable(title="Relationships")
+    power_management = SummaryTable(title="Power Management")
+    assets = SummaryTable(title="Assets")
+    firmware = SummaryTable(title="Firmware")
+    network_devices = SummaryTable(title="Network Devices")
+    smart = SummaryTable(title="Smart Management")
+
+
+class PhysicalServerDetailsView(ComputePhysicalInfrastructureServersView):
+    """Main PhysicalServer details page."""
+    breadcrumb = BreadCrumb(locator='.//ol[@class="breadcrumb"]')
+    toolbar = View.nested(PhysicalServerDetailsToolbar)
+    entities = View.nested(PhysicalServerDetailsEntities)
+
+    @property
+    def is_displayed(self):
+        title = "{name} (Summary)".format(name=self.context["object"].name)
+        return (self.in_compute_physical_infrastructure_servers and
+                self.breadcrumb.active_location == title)
+
+
+class PhysicalServerTimelinesView(TimelinesView, ComputePhysicalInfrastructureServersView):
+    """Represents a PhysicalServer Timelines page."""
+
+    @property
+    def is_displayed(self):
+        return (self.in_compute_physical_infrastructure_servers and
+                super(TimelinesView, self).is_displayed)
+
+
+class PhysicalServerManagePoliciesView(BaseLoggedInPage):
+    """PhysicalServer's Manage Policies view."""
+    policies = BootstrapTreeview("protectbox")
+    entities = View.nested(BaseNonInteractiveEntitiesView)
+    save_button = Button("Save")
```

Comment:
**Required**

We're trying to get rid of implementations with `_button` suffix, one of those cases where some code is a good example and some isn't. :)

Just `save`, `reset`, `cancel` names for these button widgets, please.